clients/python-apiserver-client/python_apiserver_client/params/headnode.py

```diff
@@ -0,0 +1,126 @@
+from .volumes import *
+from .environmentvariables import *
+
+DEFAULT_HEAD_START_PARAMS = {"dashboard-host": "0.0.0.0", "metrics-export-port": "8080"}
+
+
+class ServiceType(enum.Enum):
```

Comment:
I don't understand the connection between `ServiceType` vs. `ClusterIP` and `NodePort`.

Reply:
ClusterIP, Loadbalancer and NodePort are supported Service types